inst/data_scripts/update_mh.R

```diff
@@ -0,0 +1,56 @@
+# Add new variables to MH
+library(dplyr)
+library(tidyselect)
+library(labelled)
```

Comment:
Do we need labelled package?  I think add_labels() is from metatools.  Could just drop this package.

Reply:
Thanks, actually I think we only need the metatools package...